docs/index.rst

```diff
@@ -30,10 +30,10 @@ Dive in!
    :maxdepth: 1
    :caption: Ease of Use with a Real Robot
    
-   Switching between the Real Robot and the Simulated Robot in this Simulation <simreal/simwithreal>
-   Switching between the Real Robot and the Simulated Robot in the Real Robot API <simreal/realwithsim>
-   Switching between Non-real-time and Real-time Behaviour <simreal/simvsreal>
-   "Real" and "Non-real" TimeSteps <simreal/timesteplogic>
+   simreal/simwithreal>
```

Comment:
trailing `>`

Reply:
oh, this is an old change, i have already fixed this